src/gui/MixerLine.cpp

```diff
@@ -430,36 +430,34 @@ void MixerLine::setStrokeInnerInactive( const QColor & c )
 	m_strokeInnerInactive = c;
 }
 
-
 // Ask user for a color, and set it as the mixer line color
 void MixerLine::selectColor()
 {
 	auto channel = Engine::mixer()->mixerChannel( m_channelIndex );
-	auto new_color = ColorChooser(this).withPalette(ColorChooser::Palette::Mixer)->getColor(channel->m_color.value_or(backgroundActive().color()));
-	if(!new_color.isValid()) { return; }
-	channel->setColor (new_color);
+	auto new_color = ColorChooser(this)
+		.withPalette(ColorChooser::Palette::Mixer)
+		->getColor(channel->color().value_or(backgroundActive().color()));
+	if (!new_color.isValid()) { return; }
+	channel->setColor(new_color);
 	Engine::getSong()->setModified();
 	update();
 }
 
-
 // Disable the usage of color on this mixer line
 void MixerLine::resetColor()
 {
-	Engine::mixer()->mixerChannel(m_channelIndex)->m_color = std::nullopt;
+	Engine::mixer()->mixerChannel(m_channelIndex)->setColor(std::nullopt);
 	Engine::getSong()->setModified();
 	update();
 }
 
-
 // Pick a random color from the mixer palette and set it as our color
 void MixerLine::randomizeColor()
 {
 	auto channel = Engine::mixer()->mixerChannel( m_channelIndex );
-	channel->setColor (ColorChooser::getPalette(ColorChooser::Palette::Mixer)[rand() % 48]);
+	channel->setColor(ColorChooser::getPalette(ColorChooser::Palette::Mixer)[rand() % 48]);
```

Comment:
```suggestion
	channel->setColor(ColorChooser::getPalette(ColorChooser::Palette::Mixer)[std::rand() % 48]);
```
Use `std::` namespace prefix